ruby_on_rails/app_initialisation.md

```diff
@@ -51,7 +61,6 @@ We use a script for that so that we can change the commands for tests in a PR, w
 (usually `bundle exec figaro install` is enough)
 * and create `config/application.example.yml` where you will specify the only environment variable you need for now:
   `SECRET_KEY_BASE`.
-* Add `/config/application.yml` to your `.gitignore`
```

Comment:
done by figaro already

Reply:
I agree with `dotenv`. I don't like secrets file because it goes against the principle of 12 factor app https://12factor.net/config. I'll open an issue regarding this specific point, to discuss if we want to migrate to dotenv: #151